homeassistant/components/nuki/lock.py

```diff
@@ -165,3 +171,15 @@ def open(self, **kwargs):
 
     def lock_n_go(self, unlatch):
         """Stub service."""
+
+    def set_continuous_mode(self, enable):
```

Comment:
Is it possible to know if the mode is set to continuous? thinking a switch may be nicer than service if you can represent the state?

Reply:
This is part of my previous PR #49557. I'll defer to @pvizeli as to whether he wants to write it as a switch

Reply:
We currently not have the `entity config` finished, but once that is in place, options can be exposed as entities. You can do a switch, but then the entities need to be disabled by default. Like you want